src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentFactory.java

```diff
@@ -66,7 +68,7 @@ public RemoteAgent start(Launcher launcher, final TaskListener listener) throws
         return launcher.getChannel().call(new MinaRemoteAgentStarter(listener));
     }
 
-    private static class TomcatNativeInstalled implements Callable<Boolean, Throwable> {
+    private static class TomcatNativeInstalled extends MasterToSlaveCallable<Boolean, Throwable> {
```

Comment:
needs a `serialVersionUID` (value should be generated from the .class file before this PR)